controller/apierror/messages.go

```diff
@@ -194,4 +194,8 @@ const (
 	EnrollmentExistsCode    string = "ENROLLMENT_EXISTS"
 	EnrollmentExistsMessage string = "ENROLLMENT_EXISTS"
 	EnrollmentExistsStatus  int    = http.StatusConflict
+
+	ServerTooManyUpdatesCode    string = "SERVER_TOO_MANY_UPDATES"
```

Comment:
It appears this rate limits any mutation command and the wording on this might be confusing to some.

"I issued a delete not an update".

It might be better to change this to "NewTooManyRequests" with the same changes to the `Code` and `Message` text. At the risk of sounding more generic at least, it won't confuse external API users who

I wold also suggest changing `ServerTooManyUpdatesMessage` not only to `ServerTooManyRequestsMessage` but also to say "Too many requests to alter state have been issues. Please slow your request rate or try again later."

Reply:
fixed